src/lib/process/process_unix.c

```diff
@@ -255,18 +255,13 @@ process_unix_exec(process_t *process)
     /* Call the requested program. */
     retval = execve(argv[0], argv, env->unixoid_environment_block);
 
-    /* If we made it here it is because execve failed :-( */
-    if (-1 == retval)
-      fprintf(stderr, "Call to execve() failed: %s", strerror(errno));
-
+    /* LCOV_EXCL_START */
```

Comment:
I think it isn't appropriate to have LCOV_EXCL_START here if we believe that this code can sometimes be reached. We should only use LCOV_EXCL for things that are untestable because they are truly unreachable.